### ISSUE TITLE
Spam of notifications if daily note/heading does not exist

### ISSUE THREAD

**Issue by franciskafieh** [franciskafieh/obsidian-list-modified#61]
to add to the issue.. the plugin is really spammy if this is the case.
I'm not able to create my daily note without it at least spamming 20 times that the header doesn't exists after starting Obsidian or switching over to Obsidian on the next date.
As I have to enter some information manually into my daily note I can't autocreate it.

_Originally posted by @MMoMM-org in https://github.com/franciskafieh/obsidian-list-modified/issues/55#issuecomment-1333608723_
      

**basically, need to re-write the notification system to match interval better and be less annoying**

**Comment by franciskafieh** [comment]
added a "state" so that the user is only warned once for every issue (resets when the app is closed)
